modules/ROOT/pages/index.adoc

```diff
@@ -41,13 +39,13 @@ When a service is called, Anypoint Service Mesh ensures a smooth request flow an
 image:service-mesh-request-flow.png[]
 
 . The client, which might be another microservice or any Kubernetes Ingress component, sends a request to the service.
-. Envoy captures the request and redirects it to the Istio-mixer.
-. The mixer adapter then calls the Anypoint Service Mesh adapter to perform policy checks and verifications.
-. The Anypoint Service Mesh adapter performs policy checks and responds back to the mixer.
-. The mixer communicates the outcome of the policy checks to the Envoy proxy.
-. When no policy violations are encountered, the request is routed to the microservice. The microservice runs the service logic and sends the response back to the client.
-. The adapter communicates with Anypoint Platform asynchronously to get the latest policies and contracts.
-. Finally, the adapter sends back the API analytics information to Anypoint Platform.
+. Envoy captures the request and redirects it to the Mule adapter using the Envoy filter.
++
+The adapter then performs policy checks and verifications.
+. When no policy violations are encountered, the request is routed to the microservice. 
+. The microservice runs the service logic and sends the response back to the client.
+. Periodically, the adapter communicates with Anypoint Platform asynchronously to get the latest policies and contracts.
+. Periodically, the adapter sends back the API analytics information to Anypoint Platform.
```

Comment:
Why is this formatted as a task intro and steps?

Reply:
"sends back" > "returns"

Reply:
Because they are number in the service flow diagram and I'm explaining what happens at each step.

Reply:
Done! Thanks!